grid2op/tests/test_score_idf_2023.py

```diff
@@ -107,45 +111,65 @@ def test_score_helper(self):
             res_agent2 = my_score.get(CurtailTrackerAgent(self.env.action_space,
                                                         gen_renewable = self.env.gen_renewable,
                                                         gen_pmax=self.env.gen_pmax,
                                                         curtail_level = 0.8))
-            assert np.allclose(res_agent2[0][0][2], -127.62213025108333)
-            assert np.allclose(res_agent2[0][1][2], -143.83405253996978)
+            assert np.allclose(res_agent2[0][0][2], -120.71966085513259)
             # decrease
             assert 100. - res_agent1[0][0][2] >= res_agent1[0][0][2] - res_agent2[0][0][2]
-            assert 100. - res_agent1[0][1][2] >= res_agent1[0][1][2] - res_agent2[0][1][2]
             
             res_agent3 = my_score.get(CurtailTrackerAgent(self.env.action_space,
                                                         gen_renewable = self.env.gen_renewable,
                                                         gen_pmax=self.env.gen_pmax,
                                                         curtail_level = 0.7))
-            assert np.allclose(res_agent3[0][0][2], -169.9519401162611)
-            assert np.allclose(res_agent3[0][1][2], -179.45065441917586)
+            assert np.allclose(res_agent3[0][0][2], -164.20345819832642)
             assert res_agent1[0][0][2] - res_agent2[0][0][2] >= res_agent2[0][0][2] - res_agent2[0][0][2]
-            assert res_agent1[0][1][2] - res_agent2[0][1][2] >= res_agent2[0][1][2] - res_agent2[0][1][2]
         finally:
             my_score.clear_all() 
+            
+    def test_score_helper2(self):
+        """basic tests for ScoreL2RPN2023 class"""
+        self.env.reset() 
+        my_score = ScoreL2RPN2023(
+                    self.env,
+                    nb_scenario=self.nb_scenario,
+                    env_seeds=[0 for _ in range(self.nb_scenario)],
+                    agent_seeds=[0 for _ in range(self.nb_scenario)],
+                    max_step=self.max_iter,
+                    weight_op_score=0.6,
+                    weight_assistant_score=0.25,
+                    weight_nres_score=0.15,
+                    scale_nres_score=100,
+                    scale_assistant_score=100,
+                    min_nres_score=-100.,
+                    min_assistant_cost_score=-100)
+        try:
+            # test do nothing indeed gets 100.
+            res_dn = my_score.get(DoNothingAgent(self.env.action_space))
+            for scen_id, (ep_score, op_score, nres_score, assistant_score) in enumerate(res_dn[0]):
+                assert nres_score == 100.
+                assert ep_score == 0.6 * op_score + 0.15 * nres_score + 0.25 * assistant_score
+                assert assistant_score == 100. #no blackout with no disconnections
+                assert op_score == 0
+            
+                
+        finally:
+            my_score.clear_all()
 
     def test_min_score(self):
```

Comment:
the `min_assistant_cost_score` is added in the class that is tested, but is never tested. Maybe change the name of this test, and test the same kind of thins with the assistant (making sure that the score cannot go below the min)

Reply:
This was done in test_score_idf_2023_assistant.py